tests/test_aes_forward.py

```diff
@@ -33,26 +33,32 @@ def test_bs_out():
     # yapf: disable
```

Comment:
Then there should be no point at disabling yapf for these areas (note that the `yapf: enable` at the end should then also be removed)